src/Utilities/Wrapper.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
-import { Spinner } from '@patternfly/react-core';
+import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
 
 const RenderWrapper = ({ cmp: Component, hideLoader, isRbacEnabled, inventoryRef, store, ...props }) => {
-    const { hasAccess, isLoading } = usePermissions('inventory', [
+    const { hasAccess } = usePermissionsWithContext([
         'inventory:*:*',
         'inventory:*:read',
         'inventory:hosts:read'
-    ], true);
+    ]);
     return (
-        (isLoading && !hideLoader) ?
```

Comment:
I have a very superficial understanding of the code, but I immediately had a question if this is desired to remove the spinner and the loading state from the component logic...

Reply:
It would not make much sense here, as it would not be rendered since, the loading state for the permissions is now implemented in the `RBACProvider`. https://github.com/RedHatInsights/frontend-components/blob/88bc536dc7284618d96eb81d20ac4a3441d66ec4/packages/components/src/RBACProvider/RBACProvider.tsx#L50